packages/cluster-ui/src/statementsTable/statementsTable.module.scss

```diff
@@ -1,4 +1,4 @@
-@import 'src/core/index.module';
+@import "src/core/index.module";
```

Comment:
is there a linter we can add for this?

Reply:
I'm using [Prettier](https://prettier.io/) which I thought we had already included in [@cockroachlabs/eslint-config](https://github.com/cockroachdb/ui/blob/master/packages/eslint-config/index.js). It seems like the lint task that runs in the npm script doesn't consider css files, so that's something we might change. I think I made the assumption that everyone uses linter integrated into their IDEs, but perhaps we should move toward making linting errors break the build. 

The issue there is that there are currently hundreds of linting errors in the `cluster-ui` and DB Console code.

Reply:
👍🏻 didn't realize linting errors didn't break the build. I can look at that.